static/js/components/AnnotationsTable.jsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import {
+  makeStyles,
+  createMuiTheme,
+  MuiThemeProvider,
+  useTheme,
+} from "@material-ui/core/styles";
+import MUIDataTable from "mui-datatables";
+
+import { getAnnotationValueString } from "./CandidateAnnotationsList";
```

Comment:
It'd be helpful to capture in the filenames here and with CandidateAnnotationsList.jsx exactly where they're used, e.g. changing this one to `CandidatePageAnnotations` and the other to `ScanningPageCandidateAnnotations`. It's confusing to have all of these very similarly-named files.

Reply:
Good point, I'll do that.